dogs.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  <meta charset="UTF-8">
+  <title>Dogs</title>
+</head>
+<body>
+  <header>
+    <div class="flexbox">
+      <div class="left-side">
+        <img src="adopt-a-dog/adoptadog-logo.svg" alt="logo" width="600">
+        <h1>Seattle’s premiere dog adoption agency</h1>
+      </div>
+      <div class="right-side">
+        <img class="cart" src="adopt-a-dog/cart-icon.svg" alt="cart image" width="50" height="50">
+        $1234.56
+        <br>
+        <button type="button">Checkout</button>
+      </div>
+    </div>
+  </header>
+  <div class="nav-container">
+    <nav>
+      <a href="index.html">Home</a> /
+      <a href="dogs.html">Dogs</a> /
+      <a href="blogs.html">Blog</a>
+    </nav>
+  </div>
+  <h2> Here are all of our Dogs! </h2>
+  <div class="grid-container">
+    <div class="item1">
```

Comment:
All of these items are similar, you'll want to use the same class name for all of them instead of giving them a unique class. simply class="item" will do, but I would encourage you to use something more descriptive, such as dog-tile, or something